ApiTransform.ps1

```diff
@@ -69,6 +69,9 @@ $TransformConfig = [Ordered]@{
             '\[,'                                                                                                                                                                                                                   = '[';
             ',]'                                                                                                                                                                                                                    = ']';
         };
+        OverrideDefinitions = @(

```

Comment:
the idea here is that if you want to replace an entire section of the Swagger spec you could do that by specifying the path of item you wish to replace and adding a corresponding json file in `/CustomDefinitions/` directory